cordapp/src/main/kotlin/com/r3/corda/finance/obligation/client/flows/UpdatePaymentStatus.kt

```diff
@@ -0,0 +1,105 @@
+package com.r3.corda.finance.obligation.client.flows
+
+import co.paralleluniverse.fibers.Suspendable
+import com.r3.corda.finance.obligation.client.getLinearStateById
+import com.r3.corda.finance.obligation.client.resolver
+import com.r3.corda.finance.obligation.commands.ObligationCommands
+import com.r3.corda.finance.obligation.contracts.ObligationContract
+import com.r3.corda.finance.obligation.states.Obligation
+import com.r3.corda.finance.obligation.types.PaymentReference
+import com.r3.corda.finance.obligation.types.PaymentStatus
+import net.corda.core.contracts.UniqueIdentifier
+import net.corda.core.flows.*
+import net.corda.core.transactions.SignedTransaction
+import net.corda.core.transactions.TransactionBuilder
+import net.corda.core.transactions.WireTransaction
+import net.corda.core.utilities.ProgressTracker
+
+object UpdatePaymentStatus {
```

Comment:
maybe we can rename this to something like `update payment status manually`?

Reply:
It should also go in the manual module you added.

Reply:
Will do.